Lindsay dev

LindsayWilde opened the pull request:
This mostly has has stuff related to getting popups for inputting data with sweetalert2 and some mock data. I don't know if my returns on the HttpPosts are quite right since I wasn't sure on what to return there. The next step will probably be more mock data or just going forward and doing some actual queries with the new models to display the data.

Some screenshots of what I did
![image](https://user-images.githubusercontent.com/40272347/189263443-ac872de8-d943-4632-82a5-82e0b1b3c687.png)
![image](https://user-images.githubusercontent.com/40272347/189263453-8cb42190-0a0b-4f21-b30f-68984b84890c.png)
![image](https://user-images.githubusercontent.com/40272347/189263461-520aeac2-4098-4134-9735-c125833e8338.png)
![image](https://user-images.githubusercontent.com/40272347/189263474-314d40fd-5299-4f9f-89a1-104d3940f1c9.png)
![image](https://user-images.githubusercontent.com/40272347/189263480-0253a7c1-6056-49b6-8bfc-7faa473be2d9.png)
![image](https://user-images.githubusercontent.com/40272347/189263488-c14688df-bc7f-4aac-909a-19325c50d9a6.png)
![image](https://user-images.githubusercontent.com/40272347/189263503-d1e8ce14-d681-497e-b897-4f53f0e613d6.png)
![image](https://user-images.githubusercontent.com/40272347/189263508-ef7bfe25-5f86-4ff8-83dd-2bfbc04f907a.png)
![image](https://user-images.githubusercontent.com/40272347/189263514-b60f69a3-5063-480c-b3a7-8ac1cbe4b4b5.png)

LindsayWilde committed:
Idea of how to do things in the API controller

LindsayWilde committed:
Can display hard-coded user events, remove events and add basic events in api(somewhat) and ui

LindsayWilde committed:
Added mock data for groups and friends to see how displaying works and what data the api calls need

LindsayWilde committed:
Got sweetalert2 working for event creation for title input only

LindsayWilde committed:
Outline for sweetalert in event creation started looking into create group pop and put sample popup there

LindsayWilde committed:
sweetalert2 added to friends&groups with mock data and for the interaction with the system

The next step is do mess with mock data more and maybe try doing some actual queries to see how things will work. Such as how to get user data/id from frontend to backend

hopefully-purple commented on Let-s-Meet/Views/Group/Index.cshtml:
```diff
@@ -0,0 +1,76 @@
+    <script>
+
+        document.addEventListener('DOMContentLoaded', function () {
+            var calendarEl = document.getElementById('calendar');
```
What is calendarEl? Is "El" for "Element"?

hopefully-purple commented:
Do you mind screenshotting the ui changes you made real fast?

LindsayWilde committed:
Able to user add events via postman and get events via postman(getting might need to be changed for full calendar)

LindsayWilde committed:
able to delete user event via postman

The next steps are to make it so it connects to the identity user model not just our user model and get the getuserevents to return the json in the correct form

LindsayWilde committed:
Able to add,delete,get user events on postman with the seeded usermodel

Now need to connect this to the identity user model

LindsayWilde committed:
Front end will add, display, remove for a hard coded userID

Next step is getting rid of hard coded userID and connecting to identity

LindsayWilde committed:
Logged in users can edit their personal calendar

The event creation popup doesn't take any data, but the event name. I hope to make this popup more functional in the future now that it's working to create events

LindsayWilde committed:
worked on event creation popup

The event popup should be looking better, it will autofill based on user selection and the user can edit it after clicking and dragging their selection.
Things should run a bit more smoothly, some of the api calls might need to change based on merging branches

LindsayWilde committed:
Minor event popup bug fix and style changes

the issue is dates like 09/09/2022 didn't appear, but that should be fixed

LindsayWilde committed:
fix reloading problem with calendar and some styling

Events now display without reloading. and some style fixes. Also removed minimum date to create event

LindsayWilde commented:
I closed this one since I have a newer version that I want to put to main instead
